open-metadata-implementation/access-services/asset-lineage/asset-lineage-spring/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/server/spring/AssetLineageResource.java

```diff
@@ -5,18 +5,33 @@
 import io.swagger.v3.oas.annotations.ExternalDocumentation;
 import io.swagger.v3.oas.annotations.tags.Tag;
 import org.odpi.openmetadata.accessservices.assetlineage.server.AssetLineageRestServices;
+import org.odpi.openmetadata.commonservices.ffdc.rest.GUIDListResponse;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 
 @RestController
 @RequestMapping("/servers/{serverName}/open-metadata/access-services/asset-lineage/users/{userId}/")
 
-@Tag(name="Asset Lineage OMAS", description="The Asset Lineage OMAS provides services to query the lineage of business terms and data assets.", externalDocs=@ExternalDocumentation(description="Asset Lineage Open Metadata Access Service (OMAS)",url="https://egeria.odpi.org/open-metadata-implementation/access-services/asset-lineage/"))
+@Tag(name = "Asset Lineage OMAS", description = "The Asset Lineage OMAS provides services to query the lineage of business terms and data assets.", externalDocs = @ExternalDocumentation(description = "Asset Lineage Open Metadata Access Service (OMAS)", url = "https://egeria.odpi.org/open-metadata-implementation/access-services/asset-lineage/"))
 public class AssetLineageResource {
 
     private final AssetLineageRestServices restAPI = new AssetLineageRestServices();
 
-
-
+    /**
+     * Scan the cohort based on the given entity type
+     *
+     * @param serverName name of server instance to call
+     * @param userId     the name of the calling user
+     * @param entityType the name of the relationship type
+     * @return a list of unique identifiers (guids) of the available entities with the given type provided as a response
+     */
+    @GetMapping(path = "/initial-load-by-entity-type/{entityType}")
```

Comment:
Generally rest api focusses on a noun (the resource) and a verb - only convention, but just stands out as a slightly odd name (minor)

Reply:
I agree that we can improve the action name for this endpoint. However, this is not regular get resource endpoint that returns result but a trigger to a processing operation that will in turn publish the results as events. It is more like action command endpoint. In its nature it is more close to async call but I am not sure if this is important for the naming/design convention. In absence of @danielaotelea  I suggest to change the name to something like `/triggerInitialLoadByType/{entityType}` or `/initialLoadByType/{entityType}`

Reply:
or maybe `/publishEntitiesByType/{entityType}` ?